README.md

```diff
@@ -1 +1,19 @@
 # hw01
+# Biography: Madison Martin 
+
+I usually go by Madison in school, but I also go by *Maddie* as well. 
+I like both, so feel free to call me whichever is easiest! 
+I am a MAPSS student pursuing the Computational Social Science certificate, which is how I found this course. 
+My research interests are prejudice and discrimination, based on race/ethnicity or disability, in organizational contexts like the workplace or in education. 
+During my time in this course, and over the MAPSS program in general, I am hoping to: 
+
+1. gain coding skills by being introduced to the R programming language
+
+2. utilize these new programming-focused skills in the process of creating my master's thesis project 
+
+3. expand my knowledge and understanding of the role data plays in social science
+
+In my spare time, I recently have been collecting pins and learning about fashion history. 
+~~But, when we are not in a pandemic~~, I also like to sing, volunteer at animal shelters, and go to concerts (House of Blues is my **favorite** venue ever). 
+
+![here is my facebook profile pic because I am unoriginal](https://www.facebook.com/photo.php?fbid=4027310477284457&set=t.100000166331754&type=3)
```

Comment:
Part two - I clicked the link, and it did not take me to the photo.  So it did not really come up as a link either.

Reply:
The link worked for me, probably because I was logged into my Facebook account? Better approach is to store the image in the repo itself, then reference it directly. So if you called it `profile.png`, you could use `![](profile.png)` to embed it in the README